src/lsdb/catalog/dataset/healpix_dataset.py

```diff
@@ -182,6 +182,48 @@ def get_ordered_healpix_pixels(self) -> list[HealpixPixel]:
         pixels = self.get_healpix_pixels()
         return np.array(pixels)[get_pixel_argsort(pixels)]
 
+    def aggregate_column_statistics(
+        self,
+        use_default_columns: bool = True,
+        exclude_hats_columns: bool = True,
+        exclude_columns: list[str] | None = None,
+        include_columns: list[str] | None = None,
+        include_pixels: list[HealpixPixel] | None = None,
+    ) -> list[HealpixPixel]:
+        """Read footer statistics in parquet metadata, and report on global min/max values."""
+        if use_default_columns and include_columns is None:
+            include_columns = self.hc_structure.catalog_info.default_columns
+
+        return self.hc_structure.aggregate_column_statistics(
+            exclude_hats_columns=exclude_hats_columns,
+            exclude_columns=exclude_columns,
+            include_columns=include_columns,
+            include_pixels=include_pixels,
+        )
+
+    def per_pixel_statistics(
+        self,
+        use_default_columns: bool = True,
+        exclude_hats_columns: bool = True,
+        exclude_columns: list[str] | None = None,
+        include_columns: list[str] | None = None,
+        include_stats: list[str] | None = None,
+        multiindex=False,
```

Comment:
```suggestion
        multi_index=False,
```
Minor suggestion.  If this is already the way this term is spelled elsewhere, feel free to ignore.

Reply:
Changed in HATS in https://github.com/astronomy-commons/hats/pull/479